src/createBrowserLikeFetch.js

```diff
@@ -58,10 +59,15 @@ function createBrowserLikeFetch({
           cookieStrings.forEach((cookieString) => {
             const cookie = parse(cookieString);
             const { key, value: valueRaw, ...cookieOptions } = cookie.toJSON();
+            if (!cookieOptions.domain) {
+              // "If omitted, defaults to the host of the current document URL, not including
+              // subdomains."
+              // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#attributes
+              cookieOptions.domain = getPublicSuffix(new URL(path).host);
```

Comment:
erg, bug: [host includes the hostname and port](https://nodejs.org/docs/latest-v14.x/api/url.html#url_url_strings_and_url_objects) but `getPublicSuffix` expects only the hostname
```
$ node          
Welcome to Node.js v14.16.0.
Type ".help" for more information.
> require('tough-cookie').getPublicSuffix('www.example.com:80')
null
> require('tough-cookie').getPublicSuffix('www.example.com')
'example.com'
> 
```

```suggestion
              // host includes the hostname and port but getPublicSuffix expects only the hostname
              cookieOptions.domain = getPublicSuffix(new URL(path).hostname);
```